eng/pipelines/templates/steps/java-publishing.yml

```diff
@@ -0,0 +1,31 @@
+parameters:
+  ArtifactDirectory: not-specified
+  RepositoryUrl: not-specified
+  GroupID: not-specified
+  ArtifactID: not-specified
+  BuildToolsPath: ''
+
+steps:
+  - ${{if eq(parameters.Target, 'Burner')}}:
+    - pwsh: |
+        ${{parameters.BuildToolsPath}}/scripts/Publish-MavenPackages.ps1 -ArtifactDirectory ${{parameters.ArtifactDirectory}} -GroupIDFilter ${{parameters.GroupID}} -ArtifactIDFilter ${{parameters.ArtifactID}} -RepositoryUrl https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-$(Build.BuildID)/maven/v1 -RepositoryUsername nobody -RepositoryPassword $(System.AccessToken) -GPGExecutablePath ${{parameters.BuildToolsPath}}/tools/gpg/bin/gpg.exe -InformationAction Continue
```

Comment:
Why wouldn't the Publish-MavenPackage.ps1 also be moved out into this repo?

Reply:
I guess it could also be moved. It was not marked on the epic to be moved. The only thing that looks sensitive is the $sonaTypeProfileID but I guess that's fine to be public still.

Reply:
Yeah ... can be moved.